gammapy/estimators/flux_map.py

```diff
@@ -0,0 +1,480 @@
+    "likelihood": ["norm_err", "norm_errn", "norm_errp","norm_ul", "norm_scan", "stat_scan"],
+}
+
+class FluxMap:
+    """A flux map container.
+
+    It contains a set of `~gammapy.maps.Map` objects that store the estimated flux as a function of energy as well as
+    associated quantities (typically errors, upper limits, delta TS and possibly raw quantities such counts,
+    excesses etc). It also contains a reference model to convert the flux values in different formats. Usually, this
+    should be the model used to produce the flux map.
+
+    The associated map geometry can use a `RegionGeom` to store the equivalent of flux points, or a `WcsGeom`/`HpxGeom`
+    to store an energy dependent flux map.
+
+    The container relies internally on the 'Likelihood' SED type defined in :ref:`gadf:flux-points`
+    and offers convenience properties to convert to other flux formats, namely:
+    ``dnde``, ``flux``, ``eflux`` or ``e2dnde``. The conversion is done according to the reference model spectral shape.
+
+    Parameters
+    ----------
+    dnde_ref : `~gammapy.maps.Map`
+        the reference spectrum value
+    norm : `~gammapy.maps.Map`
+        the norm factor
+    norm_err : `~gammapy.maps.Map`, optional
+        the error on the norm factor. Default is None.
+    norm_errn : `~gammapy.maps.Map`, optional
+        the negative error on the norm factor. Default is None.
+    norm_errp : `~gammapy.maps.Map`, optional
+        the positive error on the norm factor. Default is None.
+    norm_ul : `~gammapy.maps.Map`, optional
+        the upper limit on the norm factor. Default is None.
+    norm_scan : `~gammapy.maps.Map`, optional
+        the norm values of the test statistic scan. Default is None.
+    stat_scan : `~gammapy.maps.Map`, optional
+        the test statistic scan values. Default is None.
+    ts : `~gammapy.maps.Map`, optional
+        the delta TS associated with the flux value. Default is None.
+    counts : `~gammapy.maps.Map`, optional
+        the number counts value. Default is None.
+    ref_model : `~gammapy.modeling.models.SkyModel`, optional
+        the reference model to use for conversions. Default in None.
+        If None, a model consisting of a point source with a power law spectrum of index 2 is assumed.
+
+    """
+    def __init__(
+        self,
+        dnde_ref,
+        norm,
+        norm_err=None,
+        norm_errn=None,
+        norm_errp=None,
+        norm_ul=None,
+        norm_scan=None,
+        stat_scan=None,
+        ts=None,
+        counts=None,
+        ref_model=None,
+    ):
+        self.geom = norm.geom
+        self.dnde_ref = dnde_ref
+        self.norm = norm
+        self.norm_err = norm_err
+        self.norm_errn = norm_errn
+        self.norm_errp = norm_errp
+        self.norm_ul = norm_ul
+        self.norm_scan = norm_scan
+        self.stat_scan = stat_scan
+        self.ts = ts
+        self.counts = counts
```

Comment:
Here I'm not a 100% whether we should include this information on the raw data, if we do so, I guess it means we have to include things like `excess` and `npred` as well. However these are then typically duplicated information from the `MapDataset` (with adjusted energy binning...)

Reply:
For now all non flux maps are kept within the dictionary but are not accessible via a specific property.
They are serialized along the others as well.